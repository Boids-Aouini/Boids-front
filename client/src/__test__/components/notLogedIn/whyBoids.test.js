import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import configureStore from 'redux-mock-store';
import WhyBoids from '../../../components/notLogedIn/whyBoids';

let mockStore = configureStore();
let initState = {};
let store = mockStore(initState);

describe('<WhyBoids />', () => {
    describe('render()', () => {
        test('render without crashing', () => {
            shallow(<WhyBoids store={store} />)
        })
    })

})
