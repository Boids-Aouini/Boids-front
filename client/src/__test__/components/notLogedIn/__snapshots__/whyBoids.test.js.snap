// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WhyBoids /> render() render component 1`] = `
<WhyBoids
  dispatch={[Function]}
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
/>
`;
